JS/models/classificationLevel.js

```diff
@@ -2,7 +2,7 @@ const ClassificationLevel  = {
     UNCLASSIFIED: 'Unclassified',
```

Comment:
The same as for comment regarding Experimantal Types.

Reply:
All values changed due to CAPITAL_LETTERS notation